src/app-config.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
We need to remove this envirmental data contained file it's not good idea, we can have .env files and put all this variables there.

Reply:
ok, i will remove this file